lib/map.jsx

```diff
@@ -0,0 +1,27 @@
+var React = require('react');
+
+var MapComponent = React.createClass({
+
+    propTypes: {
+      accessToken: React.PropTypes.string.isRequired,
+      mapId: React.PropTypes.string.isRequired,
+    },
+
+    componentDidMount: function () {
+      // Create a map in the div #map
+      this.map = L.mapbox.map('map', this.props.mapId, {accessToken: this.props.accessToken});
+    },
+    componentWillUnmount: function() {
+      delete L.mapbox.accessToken;
```

Comment:
Hmm, I don't think there's a need to delete `L.mapbox.accessToken` since we're not touching it anymore...